packages/web-components/fast-foundation/src/button/button.ts

```diff
@@ -3,6 +3,12 @@ import { ARIAGlobalStatesAndProperties, StartEnd } from "../patterns/index";
 import { applyMixins } from "../utilities/apply-mixins";
 import { FormAssociatedButton } from "./button.form-associated";
 
+declare global {
+    interface ShadowRoot {
+        delegatesFocus: boolean;
+    }
+}
+
```

Comment:
Do we need to declare this on the global? I think this likely exists already...

Reply:
It doesn't, at least my build was breaking without this, is there anywhere else I can add this?

Reply:
Sure, I see this having not been resolved by TS either yet